lib/actors/examples/mandelbrot/mandelbrot.nit

```diff
@@ -0,0 +1,121 @@
+# This file is part of NIT (http://www.nitlanguage.org).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example implemented from "The computer Language Benchmarks Game" - Mandelbrot
+# http://benchmarksgame.alioth.debian.org/
+#
+# Complete description of mandelbrot :
+# https://benchmarksgame.alioth.debian.org/u64q/mandelbrot-description.html#mandelbrot
+module mandelbrot is no_warning("missing-doc")
+
+import actors
+
+class Worker
+	actor
+
+	fun get_byte(x, y: Int): Int do
+		var res = 0
+		for i in [0..8[.step(2) do
+			var zr1 = crb[x + i]
+			var zi1 = cib[y]
+
+			var zr2 = crb [x + i + 1]
+			var zi2 = cib[y]
+
+			var b = 0
+			for j in [0..49[ do
```

Comment:
c'est le nombre d'itérations qu'on effectue pour déterminer si le nombre appartient à l'ensemble de mandelbrot, je vais le mettre en constante symbolique